docs/log-in-as-po.md

```diff
@@ -1,21 +1,28 @@
 # Log in as public officer
 
-1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in with TechPass**.
-<kbd>![log-in-with-techpass](assets/images/access-sgts-services-using-techpass/first.png)</kbd>
-2. Sign in to your TechPass account or choose your TechPass account.
+1. Go to the [Docs portal](https://docs.developer.tech.gov.sg/) and click **Log in**.
+
+<kbd>![new-docs](assets/images/onboarding/po-non-se/new-docs-portal.png)</kbd>
+
+2. Click **Log in with TechPass**.
+
+<kbd>![new-mfa](assets/images/onboarding/po-non-se/new-docs-tech.png)</kbd>
+
+3. Sign in to your TechPass account or choose your TechPass account.
+
 <kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
 
 If you are signing in from your GMD, you will be prompted to authenticate your WOG account.
```

Comment:
@JLeeChan Should this statement be included in Step 3 , as step 3 is applicable only if you are accessing a resource behind TechPass using your GMD?